### DISCUSSION TITLE
How to download all source strings that are unfinished in a given target language

### DISCUSSION THREAD

**Discussion by mtotschnig** [WeblateOrg/weblate#10583] [Q&A]
I would like to generate a file that holds all strings from the source language that have not yet been translated in a certain target language. My idea is to send this file to a translation service that would not want to use the Weblate interface. 
On the level of Project / Component / Target language, the Files -> Customize Download menu entry gives me the option "Unfinished strings, converted files enriched with comments; suitable for offline translation ", but the result are either empty strings (if untranslated), or the current translation (if marked for edit). How would I generate the corresponding file with the source strings, that an offline translator would need?

**Comment by nijel** [comment]
You need to choose a bilingual format in the list for that like Gettext PO or XLIFF.
